src/mgr/MetricCollector.cc

```diff
@@ -99,7 +104,7 @@ int MetricCollector<Query, Limit, Key, Report>::remove_query(MetricQueryID query
 
 template <typename Query, typename Limit, typename Key, typename Report>
 void MetricCollector<Query, Limit, Key, Report>::remove_all_queries() {
-  dout(20) << dendl;
+  dout(20) << "removing all queries" << dendl;
```

Comment:
any reason to change the log message?

Reply:
There is no logging there. So I put the log text.

Reply:
`dout(20) << dendl;` does. See `dout_prefix` in MetricCollect.cc source.